beluga_example/launch/utils/localization_launch.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 2023 Ekumen, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+
+from ament_index_python.packages import get_package_share_directory
+
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.actions import GroupAction
+from launch_ros.actions import ComposableNodeContainer
+from launch_ros.actions import Node
+from launch_ros.descriptions.composable_node import ComposableNode
+from launch.utilities.type_utils import get_typed_value
+
+from beluga_example.launch_utils import with_launch_arguments
+from beluga_example.launch_utils import (
+    get_node_with_arguments_declared_from_params_file,
+)
+
+
+def get_launch_arguments():
+    example_dir_path = Path(get_package_share_directory('beluga_example'))
+    return [
+        DeclareLaunchArgument(
+            name='localization_package',
+            default_value='beluga_amcl',
+            description='Package that provides the localization node to launch.',
+            choices=['beluga_amcl', 'nav2_amcl'],
+        ),
+        DeclareLaunchArgument(
+            name='localization_node',
+            default_value='amcl_node',
+            description='Localization node to launch.',
+        ),
+        DeclareLaunchArgument(
+            name='localization_plugin',
+            default_value='beluga_amcl::AmclNode',
+            description='Localization node to plugin to use if composition is enabled.',
```

Comment:
```suggestion
            description='Localization node plugin to use if composition is enabled.',
```

Reply:
Fixed in https://github.com/Ekumen-OS/beluga/pull/211/commits/6e66c09ad546284f94dcecc207b950c221b52347.